.github/workflows/ci_tests_dev.yaml

```diff
@@ -81,8 +84,11 @@ jobs:
       - name: Install dependencies
         run: |
           conda install ninja cmake libblas libcblas liblapack fftw gdal \
-                        ghostscript libnetcdf hdf5 zlib curl pcre ipython \
-                        dvc pytest pytest-cov pytest-mpl
+                        ghostscript libnetcdf hdf5 zlib curl pcre \
+                        numpy pandas xarray netCDF4 packaging \
+                        dvc ipython make \
+                        pytest-cov pytest-mpl pytest>=6.0 \
+                        sphinx-gallery
```

Comment:
> But perhaps you have a good reason to use conda-forge instead?

Just want to minimize the differences of these two environments, but I agree that testing `pip` is not a bad idea.

Reply:
But note that numpy is still from conda-forge, perhaps because another package depends on it.

Reply:
> But note that numpy is still from conda-forge, perhaps because another package depends on it.

Yeah, [`pytest-mpl`](https://anaconda.org/conda-forge/pytest-mpl) requires [`matplotlib-base`](https://anaconda.org/conda-forge/matplotlib-base) requires [`numpy`](https://anaconda.org/conda-forge/numpy). Do you think we should move these Python dependencies to the `pip install` line or keep them here?

Reply:
Yes, using `pip` to install all Python packages sounds a good idea.